From: Alec Leamas <leamas.alec@gmail.com>
Date: Thu, 30 Dec 2021 22:47:33 +0100
Subject: Fix wrong desktop filename in appdata

Warnings from debian checks -- the GUI install is not able
to start the program.

Forwarded: https://github.com/OpenCPN/OpenCPN/pull/2505
---
 data/opencpn.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/opencpn.appdata.xml.in b/data/opencpn.appdata.xml.in
index 1bb449a..e568d7e 100644
--- a/data/opencpn.appdata.xml.in
+++ b/data/opencpn.appdata.xml.in
@@ -10,7 +10,7 @@
   <provides>
     <binary>opencpn</binary>
   </provides>
-  <launchable type="desktop-id">org.opencpn.OpenCPN.desktop</launchable>
+  <launchable type="desktop-id">opencpn.desktop</launchable>
   <description>
     <p>
       A cross-platform ship-borne GUI navigation application made
